src/hub.js

```diff
@@ -330,11 +330,9 @@ const onReady = async () => {
 
       document.addEventListener("drop", e => {
         e.preventDefault();
-        const imgUrl = e.dataTransfer.getData("url");
-        if (imgUrl) {
-          console.log("Dropped: ", imgUrl);
-          spawnMediaInfrontOfPlayer(imgUrl);
-        }
+        const url = e.dataTransfer.getData("url");
+        const file = e.dataTransfer.files && e.dataTransfer.files[0];
```

Comment:
I think almost this same exact code will work for pasting files from the clipboard as well. We should try it real quick.